src/components/common/Options/ContextOptions.tsx

```diff
@@ -0,0 +1,34 @@
+import { ContextOptionsProps } from './ContextOptions.types';
+
+export default function ContextOptions({
+  options = [
+    { id: 1, name: '적용하기' },
+    { id: 2, name: '수정하기' },
+    { id: 3, name: '삭제하기' },
+  ],
+  onClick,
+}: ContextOptionsProps) {
+  const firstOption = 'rounded-t-lg';
+  const bar = 'border-b border-d400 mx-2';
+  const lastOption = 'rounded-b-lg';
+  return (
+    <div className="bg-d10 m-2 border border-d400 rounded-lg">
+      {options.map((option, idx) => {
+        return (
+          <>
```

Comment:
리스트를 렌더링할 때는 이렇게 key값을 넣어줘야 합니다~ 중요한 부분이니 신경써주세요~
```suggestion
      {options.map((option, idx) => {
        return (
          <Fragment key={option.id}>
```

Reply:
아래 bar를 생성하느라 상위 부모의 key값을 까먹고 있었네요. 확인했습니다 !